### ISSUE TITLE
v1.9.1 Forward Proxy mode doesn't work on tls request.(--client-proxy)

### ISSUE THREAD

**Issue by Buddle** [nghttp2/nghttp2#558]
> Browser <-- (HTTP/2, SPDY, HTTP/1.1) --> nghttpx (client-proxy mode)<=====(h2)====>nghttpx2 HTTP/2 Proxy(known as spdyproxy ) ---->squid  

```
http2-proxy=yes
frontend=*,8080;no-tls
backend=server,port;;proto=h2;tls
```

Chrome --proxy-server="http://client:8080"
http connect can work but tls ones give error " too long to response" after Chrome is showing "establishing encrypted connections".

Chrome --proxy-server="https://server:port"
All works OK.But as you know,many devices and applications only support traditional http proxy.

**Comment by tatsuhiro-t** [comment]
Thank you for telling us.  This was a regression in v1.9.0, and has been fixed in ff07018

**Comment by tatsuhiro-t** [comment]
v1.9.2 has the fix.
